cmd/scepclient/scepclient.go

```diff
@@ -221,6 +227,20 @@ func run(cfg runCfg) error {
 	return nil
 }
 
+// log our certs and their hashes
+func dumpCerts(logger log.Logger, certs []*x509.Certificate) {
+	for i, cert := range certs {
```

Comment:
Just a suggestion (feel free to ignore): WDYT about logging their length first so as in the next lines "idx" is a little bit more understandable.

Reply:
Changed "idx" to just be "number" and also included a count